db/migrate/20160225115638_remove_default_user_role.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveDefaultUserRole < ActiveRecord::Migration
+  def up
+    role = Role.where(:builtin => 1).first
+    return if role.nil?
+    role.filters.destroy_all
+    role.delete
```

Comment:
We have to manually destroy related filters and use `delete` on the role since the `check_deletable` callback prevents destroying builtin roles.